imports/ui/side-menu/side-menu.jsx

```diff
@@ -20,7 +21,7 @@ class SideMenu extends Component {
           <MenuItem onClick={isExternal ? undefined : () => dispatch(setDrawerState(false))}>
             <div className='flex items-center pv2 mv1'>
               <div className='w1-5 lh-title tc'>
-                <FontIcon className='material-icons' color='var(--mid-gray)'>{iconName}</FontIcon>
+                {text === 'Report' ? (<ReportIcon isFinalized />) : (<FontIcon className='material-icons' color='var(--mid-gray)'>{iconName}</FontIcon>) }
```

Comment:
I'd like a slightly more generic solution here. Maybe don't provide an "iconName" for the report's drawer item, but a render function instead. Then, in this line check which one of "iconName" or "iconRenderer" (suggested name) are defined and use appropriately.